modules/cnf-configure_for_irq_dynamic_load_balancing.adoc

```diff
@@ -0,0 +1,156 @@
+[id="configuring_for_irq_dynamic_load_balancing_{context}"]
+= Configuring a node for IRQ dynamic load balancing
+
+To configure a cluster node to handle IRQ dynamic load balancing, do the following:
+
+. Log in to the {product-title} cluster as a user with cluster-admin privileges.
+. Set the performance profile to use `apiVersion: performance.openshift.io/v2`.
+. Remove the `GloballyDisableIrqLoadBalancing` field or set it to `false`.
+. Set the appropriate isolated and reserved CPUs. The following snippet illustrates a profile that reserves 2 CPUs:
```

Comment:
While it is true the profile reserves 2 CPUs, what is more important is the IRQ load-balancing will be enabled for Pods running on the Isolated CPUs set. Maybe it can be emphasized here.